core/src/proxy/block_engine_stage.rs

```diff
@@ -272,25 +327,39 @@ impl BlockEngineStage {
         ),
         bundle_tx: &Sender<Vec<PacketBundle>>,
         packet_tx: &Sender<PacketBatch>,
-        trust_packets: bool,
+        block_engine_config: &BlockEngineConfig,
         verified_packet_tx: &Sender<(Vec<PacketBatch>, Option<SigverifyTracerPacketStats>)>,
         exit: &Arc<AtomicBool>,
         block_builder_fee_info: &Arc<Mutex<BlockBuilderFeeInfo>>,
+        mut auth_client: AuthServiceClient<Channel>,
+        access_token: Arc<Mutex<Token>>,
+        refresh_token: &mut Token,
+        keypair: Arc<Keypair>,
+        cluster_info: &Arc<ClusterInfo>,
+        connection_timeout: &Duration,
     ) -> crate::proxy::Result<()> {
         const METRICS_TICK: Duration = Duration::from_secs(1);
         const MAINTENANCE_TICK: Duration = Duration::from_secs(10 * 60);
-
+        // Lookahead by Maintenance Tick plus 25%
```

Comment:
unnecessary comment. what is obvious, why isn't obvious.

why = need to ensure AUTH_REFRESH_LOOKAHEAD >= MAINTENANCE_TICK. if AUTH_REFRESH_LOOKAHEAD < MAINTENANCE_TICK, then you might end up with MAINTENANCE_TICK - AUTH_REFRESH_LOOKAHEAD seconds where the access token is invalid